content/tracking/transaction_error.md

```diff
@@ -0,0 +1,209 @@
+- Where `error_description` is the longer description for the transaction error.
+- Where `error_type` is the type of error the specific issue is regarded as in the standard classification of _hard_ and _soft_. Hard error types mean the customer must provide another form of payment e.g. an expired card. Soft errors can be the result of temporary issues where retrying might be successful e.g. processor declined the transaction.
+- Where `transaction` is the transaction the issue occurred in.
+
+**Example usage:**
+
+```ts
+import { trackTransactionError } from "@snowplow/browser-plugin-snowplow-ecommerce";
+
+trackTransactionError({
+  resolution: "rejection",
+  error_code: "E123",
+  error_shortcode: "CARD_DECLINE",
+  error_description: "Card has been declined by the issuing bank.",
+  error_type: "hard",
+  transaction: {
+    revenue: 45,
+    currency: "EUR",
+    transaction_id: "T12345",
+    payment_method: "card" 
+  }
+});
+```
+
+{{% /tab %}}
+{{% tab name="JavaScript API" %}}
+
+#### `trackTransactionError`
+
+To track a transaction error you can use the `trackTransactionError` method with the following attributes:
+
+```ts
+/* {trackerName} is a placeholder for the initialized tracker on your page.  */
+
+window.snowplow("trackTransactionError:{trackerName}", {
+  resolution,
+  error_code,
+  error_shortcode,
+  error_description,
+  error_type,
+  transaction
+});
+```
+
+- Where `resolution` is the system action selected in this failure event E.g. Allowing retries, blocking the payment gateway, contacting the user or other.
+- Where `error_code` is the error-identifying code for the transaction issue E.g. E522.
+- Where `error_shortcode` is the shortcode for the transaction error E.g declined_by_api, pm_card_radarBlock.
+- Where `error_description` is the longer description for the transaction error.
+- Where `error_type` is the type of error the specific issue is regarded as in the standard classification of _hard_ and _soft_. Hard error types mean the customer must provide another form of payment e.g. an expired card. Soft errors can be the result of temporary issues where retrying might be successful e.g. processor declined the transaction.
+- Where `transaction` is the transaction the issue occurred in.
+
+**Example usage:**
+
+```ts
+window.snowplow("trackTransactionError:{trackerName}", {
+  resolution: "rejection",
+  error_code: "E123",
+  error_shortcode: "CARD_DECLINE",
+  error_description: "Card has been declined by the issuing bank.",
+  error_type: "hard",
+  transaction: {
+    revenue: 45,
+    currency: "EUR",
+    transaction_id: "T12345",
+    payment_method: "card" 
+  }
+});
+```
+
+{{% /tab %}}
+
+{{% tab name="Node.js" %}}
+#### Track transaction errors using a self describing event
+
+To track transaction errors in the Node.js tracker, you can use the `buildSelfDescribingEvents` method together with the correct parameters for the transaction error event.
```

Comment:
```suggestion
To track transaction errors in the Node.js tracker, you can use the `buildSelfDescribingEvent` method together with the correct parameters for the transaction error event.
```
I think the method is `buildSelfDescribingEvent`, if not it's wrong in the code snippet

Reply:
Yeap all fixed, thanks!